packages/wrangler/src/match-tag.ts

```diff
@@ -48,9 +49,17 @@ export async function verifyWorkerMatchesCITag(
 			throw new FatalError(
 				`The name in your ${configFileName(configPath)} file (${workerName}) must match the name of your Worker. Please update the name field in your ${configFileName(configPath)} file.`
 			);
+		} else if (e instanceof APIError) {
+			throw new FatalError(
+				"An error occurred while trying to validate that the Worker name matches what is expected by the build system.\n" +
+					e.message +
```

Comment:
will this `message` contain enough information for users to know what they need to action? Not sure we ever improved these err messages we get, but they used to be rather cryptic. If that's still the case, would be good to suggest an action on the client side the